examples/nodejs/BUILD

```
@@ -4,23 +4,30 @@
 package(default_visibility = ["//visibility:public"])
 
 load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+load("//base:distro.bzl", "DISTRO_SUFFIXES")
 
 # These examples are adapted from:
 # https://howtonode.org/hello-node
-container_image(
-    name = "hello",
-    base = "//experimental/nodejs:nodejs10",
-    cmd = ["hello.js"],
-    files = [":hello.js"],
-)
+[
+    container_image(
+        name = "hello",
+        base = "//experimental/nodejs:nodejs10" + distro_suffix,
+        cmd = ["hello.js"],
+        files = [":hello.js"],
+    )
+    for distro_suffix in DISTRO_SUFFIXES
+]
 
-container_image(
-    name = "hello_http",
-    base = "//experimental/nodejs:nodejs10",
-    cmd = ["hello_http.js"],
-    files = [":hello_http.js"],
-    ports = ["8000"],
-)
+[
+    container_image(
+        name = "hello_http",
+        base = "//experimental/nodejs:nodejs10" + distro_suffix,
```

```suggestion
        name = "hello_http_" + distro_suffix,
        base = "//experimental/nodejs:nodejs14_" + distro_suffix,
```
(The code is untested.)